src/App.tsx

```diff
@@ -1,11 +1,48 @@
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { FindMovie } from './components/FindMovie';
 import { Movie } from './types/Movie';
+import { MovieData } from './types/MovieData';
+
+import { getMovie } from './api';
 
 export const App = () => {
-  const [movies] = useState<Movie[]>([]);
+  const [movies, setMovies] = useState<Movie[]>([]);
+  const [formInput, setFormInput] = useState('');
```

Comment:
do we really need to store formInput in the App, or it can be moved to FindMovie?

Reply:
I ask because I want you to consider this point